src/pages/main/Main.tsx

```diff
@@ -1,22 +1,27 @@
 import React from 'react';
 import { useNavigate } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
-
+import Ticker from 'react-ticker';
 import 'config/i18n';
 
 import Button from 'components/button/Button';
-import routerPaths from '../../components/navigation/paths';
+import routerPaths from 'components/navigation/paths';
+import tickerVars from 'variables/ticker';
 
 import style from './Main.module.scss';
 
 function MainPage(): JSX.Element {
   const { t } = useTranslation();
   const navigate = useNavigate();
 
+  const tickerPhrases: string[] = t('ticker', { returnObjects: true });
+
+  const MemoTicker = React.memo(Ticker);
```

Comment:
Does it matter?

Reply:
it doesn't ??
okay, I should export separate component like this